uui-components/src/navigation/MainMenu/MainMenuLogo.tsx

```diff
@@ -17,12 +17,14 @@ export class MainMenuLogo extends React.Component<MainMenuLogoProps, any> {
         return (
             <div onContextMenu={ this.props.onContextMenu } onClick={ this.props.onClick } {...this.props.rawProps}>
                 <Anchor cx={ css.container } link={ this.props.link } href={ this.props.href } isDisabled={ !this.props.href && !this.props.link }>
-                    <img
-                        className={ css.logo }
-                        alt='Main Menu Logo'
-                        src={ this.props.logoUrl ? this.props.logoUrl : undefined }
-                        style={ { backgroundColor: this.props.logoBgColor } }
-                    />
+                   {this.props.logoUrl && (
```

Comment:
move this check to the mainMenu component, check it when we make the list on menu items